homework/tests/Calctest.spec.js

```diff
@@ -0,0 +1,174 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+const data = [
+  'Prototype',
+  '1',
+  '2',
+  '3',
+  '4',
+  '5',
+  '6',
+  '7',
+  '8',
+  '9'
+]
+
```

Comment:
All the selectors in the code could be described as constants if they do repeat. For example:
const firstNumberInputField = '#number1Field';

So then you can use it in code like await page.locator(firstNumberInputField ).isVisible()

Image situation if you have 2000 tests where selector is written in a way like you did, and it changes to something like '#numberXField' - now you would need to refactor code and change it in every single place instead of easily updating the selector constant.